.storybook/test-runner.ts

```
@@ -0,0 +1,26 @@
+// COMMENTED OUT for now until we want to actually turn on accessibility tests on CI
```

Do the tests pass for you locally? test-storybook passed for me (which I was surprised by, if it included accessibility tests?) but I'm not 100% confident that I resolved the merge conflicts correctly

---

We can see the accessibility failures both locally and on CI by uncommenting the contents of this file. See [this build](https://github.com/greenriver/hmis-frontend/actions/runs/11236420220/job/31236495367?pr=935), for example.

Do you want to actually turn this on now and make all our CI tests fail until we fix the a11y issues? I had assumed not yet... 

But maybe it would be a good idea to turn it on and factor it out into a separate build step, that way when it fails we can immediately recognize the failure is expected, instead of clicking into the gh action log to make sure it's failing on the expected step?

---

Huh okay I wasn't seeing them locally, but I'll try again.  No I agree, lets not intentionally have CI fail.